ui/js/login.js

```diff
@@ -0,0 +1,33 @@
+window.onload = function () {
+    const formData = document.getElementById('formData')
+    formData.addEventListener('submit', loginUser);
+}
+function loginUser(event) {
+    /*
+    Function to perform user login
+    */
+    event.preventDefault();
+
+    let username = document.getElementById('username').value
+    let password = document.getElementById('password').value
+
+    fetch('http://127.0.0.1:5000/api/v2/auth/login', {
```

Comment:
You can try to refactor the fetch statement URL.
e.g url =''http://127.0.0.1:5000/"
then 
fetch(url +'api/v2/auth/login',

This will allow you to change the URL string easily in the near future.